src/components/RichTextContainer.ts

```diff
@@ -150,15 +155,28 @@ export default class RichTextContainer extends Component<RichTextContainerProps,
         onSuccess();
     }
 
-    private executeAction(mxObject: mendix.lib.MxObject, action?: string) {
-        if (action) {
-            window.mx.ui.action(action, {
+    private executeAction(mxObject: mendix.lib.MxObject) {
+        const { onChangeMicroflow, onChangeNanoflow, mxform } = this.props;
+
+        if (onChangeMicroflow) {
+            window.mx.ui.action(onChangeMicroflow, {
                 origin: this.props.mxform,
                 params: {
                     guids: [ mxObject.getGuid() ],
                     applyto: "selection"
                 },
-                error: error => window.mx.ui.error(`Error while executing microflow: ${action}: ${error.message}`)
+                error: error => window.mx.ui.error(`Error while executing microflow: ${onChangeMicroflow}: ${error.message}`)
+            });
+        }
+
+        if (onChangeNanoflow.nanoflow) {
+            const context = new mendix.lib.MxContext();
+            context.setContext(mxObject.getEntity(), mxObject.getGuid());
+            window.mx.data.callNanoflow({
+                context,
+                error: error => window.mx.ui.error(`Error while executing nanoflow: + ${onChangeNanoflow} : ${error.message}`),
```

Comment:
Re-format error to
`window.mx.ui.error(`Error while executing nanoflow: ${onChangeNanoflow} : ${error.message}``
Plus sign not required for template strings